Content.IntegrationTests/Tests/EntityTest.cs

```diff
@@ -14,6 +14,7 @@
 namespace Content.IntegrationTests.Tests
 {
     [TestFixture]
+    [NonParallelizable, SingleThreaded, RequiresThread]
```

Comment:
Running integration tests in parallel is something that is 100% desirable in the future due to their relatively high cost currently so I feel this is a crappy bandaid at best.